### DISCUSSION TITLE
Feature Request: Dark Mode

### DISCUSSION THREAD

**Discussion by hnt3b453** [balzack/databag#24] [Ideas]
Thought of this this morning when I got a message and checked it and kinda blinded myself. I've got all my computing devices and apps that support it set to dark mode by default.

Thank you!

**Comment by balzack** [comment]
hah :)....i can see happening. Another great request!

**Reply by hnt3b453** [reply]
:) Look forward to it!

**Comment by ghost** [comment]
+1 for this feature request

**Comment by balzack** [comment]
I've been slowly aggregating wireframes for a revamped UI. Besides looking more polished it will include support for dark mode. If I dont start making faster progress, I will work on this feature separately sooner.
